### DISCUSSION TITLE
Picker does not open in CEP panel

### DISCUSSION THREAD

**Discussion by AlexMcComb** [adobe/react-spectrum#2478] [Q&A] [answered]
This might have more to do with CEP than React Spectrum, but the `Picker` dropdown does not open in my CEP panel.  Strangely it works if I have the debugger open in localhost, but if I close that localhost, the dropdown stops working.  I narrowed it down to a `div` being appended to the `body` when the dropdown is opened.  If that `div` gets appended, the dropdown works.  I highlighted it in this screenshot:

<img width="639" alt="Screen Shot 2021-10-20 at 5 06 22 PM" src="https://user-images.githubusercontent.com/16233802/138184584-d6b5772f-a35a-4ca9-ab91-79e10da2c386.png">

Is there any idea why this might not get appended?  

Here is my `Picker` code:
```
      <Picker label="Choose frequency">
        <Item key="rarely">Rarely</Item>
        <Item key="sometimes">Sometimes</Item>
        <Item key="always">Always</Item>
      </Picker>
```

**Comment by snowystinger** [comment, marked as answer]
This might be related https://github.com/adobe/react-spectrum/issues/854#issuecomment-673760082 could you take a look?

**Comment by LFDanLu** [comment]
@AlexMcComb If the above workaround that @snowystinger mentioned happens to fix the issue for you, would you mind sharing what CEP build you are using? Trying to provide some information to the CEP team for reproduction.

**Reply by AlexMcComb** [reply]
That did work for me.  We are using CEP 10.

**Reply by rushilmahajan-adobe** [reply]
HI @AlexMcComb  
CEP team is looking at this issue, please share the working workaround with us

**Reply by AlexMcComb** [reply]
In the root component I added `delete window.PointerEvent`.
